site/content/docs/5.1/content/reboot.md

```diff
@@ -58,15 +58,18 @@ $font-family-sans-serif:
   // Web font
   HelvNeueOrange,
   // Helvetica Neue
-  Helvetica Neue,
+  "Helvetica Neue",
   // Web Safe Fallbacks
-  Helvetica, Arial,
-  // Ubuntu
-  Liberation Sans,
+  Helvetica,
+  // Linux,
+  "Noto Sans",
```

Comment:
IMO, these changes seems to be fine, this font is under SIL Open Font License and according to [this site](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL), it sounds great to me. As I don't have any VM and Browserstack doesn't provide any, I didn't tried on Linux (but works as before for Windows and MacOS).